modules/admin-ui-picard/app/src/components/events/partials/wizards/StartTaskWorkflowPage.js

```diff
@@ -0,0 +1,128 @@
+import React, {useEffect, useState} from "react";
+import {useTranslation} from "react-i18next";
+import RenderWorkflowConfig from "./RenderWorkflowConfig";
+import {fetchWorkflowDef} from "../../../../thunks/workflowThunks";
+import {getWorkflowDef} from "../../../../selectors/workflowSelectors";
+import {connect} from "react-redux";
+import cn from 'classnames';
+
+/**
+ * This component renders the workflow selection for start task bulk action
+ */
+const StartTaskWorkflowPage = ({ formik, previousPage, nextPage, loadingWorkflowDef, workflowDef }) => {
+    const { t } = useTranslation();
+
+    // Cache whole selected workflow definition for further use
+    const [selectedWorkflow, setSelectedWorkflow] = useState([]);
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() => {
+        // Load workflow definitions for selecting
+        loadingWorkflowDef();
+        setLoading(false);
+    }, []);
+
+    const handleChange = e => {
+        const workflowId = e.target.value;
+        formik.setFieldValue('workflow', workflowId);
+        const chosenWorkflow = workflowDef.find(workflow => workflow.id === workflowId);
+        setSelectedWorkflow(chosenWorkflow);
+    }
+
+    const descriptionBoxStyle = {
+        margin: '15px 0 0 0',
+        position: 'relative',
+        border: 'solid #c9d0d3',
+        borderWidth: '1px',
+        backgroundColor: '#fafafa',
+        overflow: 'hidden',
+        borderRadius: '4px'
+    };
+
+    const descriptionTextStyle = {
+        margin: '10px',
+        fontFamily: '"Open sans", Helvetica,sans-serif',
+        fontSize: '12px',
+        whiteSpace: 'pre-line'
+    };
+    return (
+        <>
+            <div className="modal-content">
+                <div className="modal-body">
+                    <div className="full-col">
+                        {/* Workflow definition Selection*/}
+                        <div className="obj list-obj">
+                            <header>{t('BULK_ACTIONS.SCHEDULE_TASK.TASKS.SELECT')}</header>
+                            <div className="obj-container">
+                                {!loading && (
+                                    workflowDef.length > 0 && (
+                                        <select tabIndex="99"
+                                                name="workflow"
+                                                onChange={e => handleChange(e)}
+                                                placeholder={t('EVENTS.EVENTS.DETAILS.PUBLICATIONS.SELECT_WORKFLOW')}
+                                                style={{width: '100%'}}>
+                                            <option value="" />
```

Comment:
Is there a reason, why we want the user to be able to select an empty string in the drop-down menu? (I know, that the next-button is disabled in this case, the question is only about the user being able to select the option).
If you intended it as a non-selectable initial value, you could add `disabled` (and `hidden`)

Reply:
The placeholder isn't shown in the collapsed dropdown menu, before a value is selected. If you can't get it to work otherwise in this case, you could try using the currently empty option for this:
`<option value="" disabled hidden> {t('EVENTS.EVENTS.DETAILS.PUBLICATIONS.SELECT_WORKFLOW')} </option>`

Reply:
the empty option don't need `disabled` or `hidden` because this is handled by the validation. If the user chooses the placeholder option then the next button gets deactivated.